site/zenodo_rdm/legacy/services.py

```diff
@@ -109,8 +110,46 @@ def create(self, identity, data, uow=None, expand=False):
                 uow=uow,
             )
 
+        communities = data.get("communities", [])
```

Comment:
minor: shouldn't we actually pop this before passing it to `super().create(...)` above?